docs/source/glossary.md

```diff
@@ -0,0 +1,79 @@
+# Commonly used terms
+
+(def_actor)=
+
+## Actor
+
+Definition of an actor
+
+(def_application_component)=
+
+## Application component
+
+Definition of a application component
+
+(def_application_service)=
+
+## Application service
+
+Definition of a application service
+
+(def_architectural_principle)=
+
+## Architectural principle
+
+Fundamental guidelines that inform the design, decision making and implementation of a TRE.
+These principles provide a framework to ensure that the design of the underlying components of a TRE are aligned to consistent goals, values and best practices.
+
+(def_business_data_object)=
+
+## Business data object
+
+Definition of decomposition
+
+(def_capability)=
+
+## Capability
+
+An ability that a system possesses.
+Capabilities are typically expressed in general and high-level terms and typically require a combination of organisation, people, processes, and technology to achieve.
+
+(def_capability_decomposition)=
+
+## Capability decomposition
+
+A set of components that realise a capability.
+These components will vary depending on the nature of the capability.
+Business-focused capabilities will be realised by business processes, roles and services.
+Technology-focused capabilities will be realised by applications, application services and interfaces.
+In addition to the components realising the capability, a catalogue of standards, frameworks and controls linked to the capabilities will provide guidance on how to implement the capabilities safely.
+
+(def_component)=
+
+## Component
+
+Definition of a component
+
+(def_process)=
+
+## Process
+
+Definition of a process
+
+(def_role)=
+
+## Role
+
+Definition of a role
+
+(def_specification_pillar)=
+
+## Specification pillar
+
+Definition of a specification pillar
```

Comment:
SATRE is split into 4 specification pillars that group related capabilities:
Information governance, Computing technology, Data management and Supporting Capabilities.